src/Widgets/WindowSwitcher.vala

```diff
@@ -173,6 +182,27 @@ namespace Gala
 			dock_surface = null;
 		}
 
+		void update_background ()
```

Comment:
"wm.get_screen ().get_size (out int width, out int height);" is not working for?